### PR TITLE
Publish npm packages

### PR DESCRIPTION
**Pull request by wollooo**
see https://github.com/kumodao/borrowprot/issues/83

### CODE REVIEW

File: packages/lib-ethers/README.md [hunk 1]
```diff
@@ -14,7 +14,8 @@ Connecting to an Ethereum node and sending a transaction:
 
 ```javascript
 const { Wallet, providers } = require("ethers");
-const { EthersKumo } = require("@liquity/lib-ethers");
+const { EthersLiquity } = require("kumodao/lib-ethers");
```
Comment (udosson) [inline packages/lib-ethers/README.md:17]: Shouldn't this be `EthersKumo`?

File: packages/lib-base/package.json [hunk 2]
```diff
@@ -8,12 +8,15 @@
...
   ],
-  "homepage": "https://github.com/liquity/liquity#readme",
+  "homepage": "https://github.com/kumodao/borrowprot#readme",
   "license": "MIT",
-  "author": "Daniel Simon <dani@liquity.org>",
+  "author": "Robin Klemens <robin@kumo.earth>",
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
-  "repository": "https://github.com/liquity/dev",
+  "repository": {
```
Comment (udosson) [inline packages/lib-base/package.json:16]: What is the reason for changing the `repository {}` structure?

**Review by udosson — CHANGES_REQUESTED** [review]
@wollooo thanks for the PR. One questions and one change :)

File: packages/lib-base/package.json (hunk 2 above)
Comment (wollooo) [reply packages/lib-base/package.json:16]: i auto generated the file with `npm init --yes`

File: packages/lib-base/package.json [hunk 3]
```diff
@@ -13,10 +13,7 @@
   "author": "Robin Klemens <robin@kumo.earth>",
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/kumodao/borrowprot.git"
-  },
+  "repository": "git+https://github.com/kumodao/borrowprot.git",
```
Comment (udosson) [inline packages/lib-base/package.json:16]: Can we change this to `https://github.com/kumodao/borrowprot` only since we've removed type `git`

**Review by udosson — CHANGES_REQUESTED** [review]
Thanks, @wollooo 
Maybe one small change. What do you think?

File: packages/lib-base/package.json (hunk 3 above)
Comment (wollooo) [reply packages/lib-base/package.json]: sure should work

**Pull request merged**